controllers/api/comment-routes.js

```diff
@@ -0,0 +1,110 @@
+const router = require("express").Router();
+
+const { User, Post, Comment } = require("../../models");
+
+// Get all comments
+router.get("/", async (req, res) => {
+  try {
+    const allComments = await Comment.findAll({
+      attributes: ["id", "comment_text", "post_id", "user_id", "created_at"],
+      order: [["created_at", "DESC"]],
```

Comment:
great addition here! :-)

It'd be cool to apply something similar or reverse the array at the endpoint where you're retreieving posts from the DB to display on the homepage so that the newest appears at the top:-)